examples/emoji-color-cairo.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+#  pycairo/cairocffi-based emoji-color example - Copyright 2017 Hin-Tak Leung
+#  Distributed under the terms of the new BSD license.
+#
+#  This script demonstrates overlapping emojis.
+#
+
+import freetype
+import numpy as np
+from PIL import Image
+
+from cairo import ImageSurface, FORMAT_ARGB32, Context
+
+face = freetype.Face("Apple Color Emoji.ttf")
```

Comment:
ttc is larger and has a 2nd typeface (.e. `Face("Apple Color Emoji.ttc", 1)`). Anyway, I have added another commit t to use the full sierra path, asI think it should either work on a current and correct os, or not at all, and added a comment about older Mac Os and Linux.